pkg/pillar/hypervisor/containerd.go

```diff
@@ -231,20 +234,46 @@ func (ctx ctrdContext) Info(domainName string) (int, types.SwState, error) {
 	defer done()
 	effectiveDomainID, exit, status, err := ctx.ctrdClient.CtrContainerInfo(ctrdCtx, domainName)
 	if err != nil {
-		return 0, types.UNKNOWN, logError("containerd looking up domain %s resulted in %v", domainName, err)
+		return 0, 0, logError("containerd looking up domain %s resulted in %v", domainName, err)
```

Comment:
I don't think returning `0` as the second value makes sense, as the caller will most likely try to compare it to known `SwState` constants:
https://github.com/lf-edge/eve/blob/df51e7ab8b0acfd5a074159bc2e5200187fa1fc1/pkg/pillar/types/types.go#L24
`0` is not defined there. I think `type.UNKONW` is still the best fit.

Reply:
I agree this should remain types.UNKNOWN bit the third argument should be nil (logError returns an error). Thus log the error and return 0, types.UNKNOWN, nil.